tests/test_process_elife_zip_dag.py

```diff
@@ -202,6 +202,35 @@ def test_strip_related_article_tags_from_article_xml_using_article_without_relat
     assert return_value == article_xml
 
 
+def test_strip_object_id_tags_from_article_xml_using_article_with_related_article_tag(context):
+    # setup
+    test_asset_path = str(get_asset('elife-36842.xml').absolute())
+    article_xml = etree.parse(test_asset_path)
+    xpath = '//object-id'
+    assert len(article_xml.xpath(xpath)) == 49
```

Comment:
a bit of a magic number, could just be asserted to be greater than 0 and the test would still cover this case?

Reply:
updated